PR#4492 (updates for upcoming release) Review - suggestions

flo91 opened the pull request:
This are just some suggestions for improvements in the course of a review for the PR #4492.
As that PR was already merged, I decided to publish my suggested changes as a new PR.

**Review:**
The content of the documents looks really good. 👍🏻 
I could not find a thing that is unclear or hard to understand, esp. after including the suggestions from other reviews. 💯 
However, I made some changes like adding links, formatting or rephrasing.

Please review my suggested changes, thx!

markus2330 reviewed (approved):
Looks good to me, needs a rebase though.

kodebach commented:
Also: Merging this will (probably) cause conflicts in #4187. So I would suggest merging #4187 first and then rebasing this.

markus2330 commented:
@flo91 can you please rebase, I would like to merge it soon. In the meeting notes there is a link to the documentation guides and it would be great if most people can already read with the fixes.

flo91 commented:
Thank you for the reminder! :bell:
I almost forgot about this PR. ^^'

I've rebased with current master and added the broken links to the MinGW builds to the linkchecker whitelist. (see issue #4624)

Unfortunately, there seems to be a problem with the javadoc generation on Debian unstable, all three jobs on Debian unstable failed with the same error.
See: https://build.libelektra.org/blue/organizations/jenkins/libelektra/detail/PR-4506/7/pipeline/1093

markus2330 commented:
Thank you for updating! :rocket: 

Did you confirm that this problem is independent of this PR? (E.g. by creating an PR with only updating release notes.) Can you change that Javadoc generation will be done on Debian stable (like other docu tasks)?

If it is not so easy, could you please create an issue?

atmaxinger commented:
@markus2330 the java doc issue is also appearing in the decision PR #4554, so it seems independent from this PR.

markus2330 commented:
Please rebase after https://github.com/ElektraInitiative/libelektra/pull/4635

markus2330 commented on doc/decisions/operation_sequences.md:
```diff
@@ -0,0 +1,253 @@
+# Operation sequences
```
Something went wrong with the rebase.

flo91 replied:
Looks like it, the rebase skipped my last commit (add broken links to whitelist) because that was already done in PR #4635 and a following `git pull` messed up the branch quite a bit.

Now, it hopefully should be ok.

markus2330 commented:
Great job, thank you! :heart:
